app/src/main/java/app/grapheneos/camera/CamConfig.kt

```diff
@@ -343,6 +347,39 @@ class CamConfig(private val mActivity: MainActivity) {
             return "${SettingValues.Key.VIDEO_QUALITY}_$pf"
         }
 
+    var videoFrameRate: Range<Int> = SettingValues.Default.VIDEO_FRAME_RATE
+        get() {
+            return if (modePref.contains(videoFrameRateKey)) {
+                Log.i("TAG", "videoFrameRate from pref")
```

Comment:
`"TAG"` log tag is uninformative, it makes searching the log harder.

Reply:
Ah that log statement was only for temporary debugging purpose, should have removed it earlier. Sorry for that. Fixed this now